.github/workflows/ci.yml

```diff
@@ -84,3 +84,18 @@ jobs:
           build/sdbus-c++*.deb
           build/sdbus-c++*.ddeb
         retention-days: 10
+  freebsd-build:
+    name: build (freebsd, clang/libc++, basu)
+    runs-on: macos-12 # until https://github.com/actions/runner/issues/385
+    steps:
+    - uses: actions/checkout@v2
+    - name: Test in FreeBSD VM
+      uses: vmactions/freebsd-vm@v0
+      with:
+        copyback: false
+        usesh: true
+        prepare: |
+          pkg install -y cmake ninja pkgconf basu expat googletest
+        run: |
+          cmake -B _build -G Ninja -DBUILD_CODE_GEN=ON -DBUILD_TESTS=ON -DENABLE_PERF_TESTS=ON -DENABLE_STRESS_TESTS=ON
+          cmake --build _build
```

Comment:
- 1 integration test hangs since 1.3.0 (haven't filed a bug yet)
-  `etc/dbus-1/system.d/org.sdbuscpp.integrationtests.conf` requires system bus (dbus daemon running as root)

Reply:
Also, I didn't fix installation of tests yet: `/etc/dbus-1` doesn't respect `CMAKE_INSTALL_PREFIX` and `/opt/test/bin` doesn't respect hier(7).

Reply:
To the respective points:

1. May be an issue beyond sdbus-c++ (see https://github.com/Kistler-Group/sdbus-cpp/issues/359#issuecomment-1721959843). Or will be fixed with sdbus-c++ v2.0 (once `release/v2.0` is merged into `master`).
2. Is this really an issue? D-Bus daemon with system bus on my native system as well as embedded system runs under `messagebus` user. This is configured in `/usr/share/dbus-1/system.conf` (as per `man dbus-daemon`). Alternatively, integration tests could be modified to run on session bus. But not all systems (like the embedded one of mine) run D-Bus daemon with session bus.
3. `/opt/test/bin` is a default value of `TESTS_INSTALL_PATH` variable, and thus can be overridden. However, true, `/etc/dbus-1` is fully hardcoded absolute path. We could fix this in this PR, by removing the `/` from the beginning of those paths.

Reply:
> 1\. May be an issue beyond sdbus-c++

Or an undefined behavior in sdbus-cpp exposed by a different threading library. I'm mainly a packager, not familar with debugging threads.

> 2\. Is this really an issue?

Somewhat:
- FreeBSD doesn't ship with DBus in base system. Downstream (packaging) only allows root during installation of dependencies.
- Many DBus consumers (both general and some sdbus-cpp) only need user/session bus thus work fine without extra privileges required to run system bus.
- Services on system bus are more destructive thus not a good idea to run multiple instances or pass through socket from host to jail.
- Installing system bus configs under `etc/dbus-1/system.d` requires `dbus-daemon` if the directory didn't exist before.
- System bus isn't required on non-systemd distros: non-existent on servers, unused besides bluetooth (bluez), battery status (upower) and session tracking (elogind, consolekit2) on desktop.

Looks like `freebsd-vm` runs everything as root, so I've enabled the tests.

> 3\. `/opt/test/bin` is a default value of `TESTS_INSTALL_PATH` variable, and thus can be overridden.

This shouldn't be required by default. Fixed in d81ab4ba30a2

> We could fix this in this PR, by removing the `/` from the beginning of those paths.

This breaks Linux CI (`org.sdbuscpp.integrationtests.conf` is now unused), so fixed by b748634d1de2 + 815190361ed4.